src/query/frontend/semantic/symbol_generator.hpp

```diff
@@ -278,10 +282,7 @@ class PropertyLookupEvaluationModeVisitor : public ExpressionVisitor<void> {
 inline SymbolTable MakeSymbolTable(CypherQuery *query, const std::vector<Identifier *> &predefined_identifiers = {}) {
   SymbolTable symbol_table;
   SymbolGenerator symbol_generator(&symbol_table, predefined_identifiers);
-  query->single_query_->Accept(symbol_generator);
-  for (auto *cypher_union : query->cypher_unions_) {
```

Comment:
```
    if (visitor.PreVisit(*this)) {
      single_query_->Accept(visitor);
      for (auto *cypher_union : cypher_unions_) {
        cypher_union->Accept(visitor);
      }
    }

    return visitor.PostVisit(*this);
```

in ast.hpp, this is the accept to be called, that's why I ommitted this one, because it already exists in the ast logic.